metadata-service/auth-impl/src/main/java/com/datahub/authorization/PolicyEngine.java

```diff
@@ -126,8 +109,8 @@ public PolicyActors getMatchingActors(
 
   private boolean isPolicyApplicable(
       final DataHubPolicyInfo policy,
-      final Urn actor,
-      final Optional<ResolvedResourceSpec> resource,
+      final ResolvedEntitySpec resolvedActorSpec,
```

Comment:
actor? to keep naming consistent i'd think

Reply:
(im okay if we do in a followup)

Reply:
Our understanding is that resolvedEntitySpec is preferred as it is consistent with other parameters. However, in this case the parameter must be an actor (the corpuser that is doing the request) and we believe that the current name narrows the purpose of the parameter and helps to understand possible values as not any resolved entity spec can be received.
We hope you agree to keep the current name. In case you still think the rename is a better option we will apply the change on the next PR to avoid blocking this one 🙂